### PR TITLE
Change dietary requirements statistics panel

### PR DESCRIPTION
**Pull request by keaganpzh**
Currently, the dietary requirements panel show individual dietary requirements as separate entries, which results in inaccurate representation of dietary requirements in the guest list.

Other changes include handling shortcut commands for other operating systems, changing the height of the result display, Removing the statistics panel and putting the numbers at the titles of the lists, and adding a codecov configuration file to avoid checking UI files not covered by automated testing.

Let's:

- [x] Edit the implementation of `DietaryRequirementStatistics` to handle different permutations of dietary requirements
- [x] Handle keyboard shortcuts for other OS
- [x] Change height of result display
- [x] Remove `StatisticsPanel.*` 
- [x] Add `codecov.yml` file
- [x] Track the number of guests with no specified dietary requirements

Addresses #156 and #158.

### CODE REVIEW

**Comment by keaganpzh** [comment]
This PR is ready for review but not ready to be merged. We need to take the screen recordings for our product demo of the old UI before we make changes.

File: src/main/java/wedlog/address/ui/DietaryPanel.java [hunk 1]
```diff
@@ -39,8 +41,9 @@ public DietaryPanel(Logic logic) {
 
     private ObservableList<String> generateList() {
         ObservableList<String> dietaryList = FXCollections.observableArrayList();
-        logic.getDietaryRequirementStatistics().getMap()
-                .forEach((k, v) -> dietaryList.add(k + ": " + v));
+        logic.getDietaryRequirementStatistics().getMap().entrySet().stream()
```
Comment (tllshan) [inline src/main/java/wedlog/address/ui/DietaryPanel.java:44]: let's also include a count for "regular" meals that tallies up the guests whose d/ fields are empty

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -13,7 +13,7 @@
 <?import javafx.scene.layout.VBox?>
 <?import javafx.stage.Stage?>
 
-<fx:root minHeight="660" minWidth="900" onCloseRequest="#handleExit" title="WedLog" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17.0.7" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root minHeight="700" minWidth="900" onCloseRequest="#handleExit" title="WedLog" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17.0.7" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (tllshan) [inline src/main/resources/view/MainWindow.fxml:16]: the new min height for the app doesn't fit withing my com's screen, which causes the status bar to get cut off. perhaps we should reduce the min height by a bit?

File: src/main/java/wedlog/address/ui/MainWindow.java [hunk 3]
```diff
@@ -223,7 +218,6 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
-            setStatisticsPanel();
```
Comment (tllshan) [inline src/main/java/wedlog/address/ui/MainWindow.java:226]: the current guest and vendor counts displayed at the top of each panel don't get updated when guests/vendors are added and deleted. think we may need to have a setGuestListPanel and a setVendorListPanel method that get called  in executeCommand to update the numbers each time

**Review by tllshan — APPROVED** [review]
LGTM!

**Pull request merged**